ngx_sxg_utils.c

```diff
@@ -232,3 +235,118 @@ X509* load_x509_cert(const char* filepath) {
   fclose(certfile);
   return cert;
 }
+
+ngx_sxg_cert_chain_t ngx_sxg_empty_cert_chain() {
+  static ngx_sxg_cert_chain_t empty;
+  empty.serialized_cert_chain = sxg_empty_buffer();
+  empty.certificate = NULL;
+  empty.issuer = NULL;
+  empty.ocsp = NULL;
+  return empty;
+}
+
+void ngx_sxg_cert_chain_release(ngx_sxg_cert_chain_t* target) {
+  sxg_buffer_release(&target->serialized_cert_chain);
+  if (target->certificate != NULL) {
+    X509_free(target->certificate);
+  }
+  if (target->issuer != NULL) {
+    X509_free(target->issuer);
```

Comment:
Should this `NULL` out the pointers after freeing?

Reply:
Done.